### PR TITLE
Diameter binary tree - Python

### LINKED ISSUE #1739: Diameter of a Binary tree
**Is your feature request related to a problem? Please describe.**
To print the diameter of a given binary tree. 

**Describe the solution you'd like**
A binary tree diameter equal to the total number of nodes on the longest path between any 2 nodes in it. 

**Do you want to work on it**
Python

I am a GSSOC'21 Participant and would like to work on this issue.

### PR DESCRIPTION
**Pull request by nidhivanjare**
Code for Diameter Od a binary tree in java and python. 

Fixes #1739

This PR is for Python
PYTHON :

Screenshot 2021-04-28 at 3 18 22 PM

### CODE REVIEW

**Comment by nidhivanjare** [comment]
DONE !!

**Comment by krishna-NIT** [comment]
@nidhivanjare  only 1 issue in 1 pr
so please update it
and comment here with issue no, like Fixes #<your issue number>

**Comment by nidhivanjare** [comment]
added the issue no in the comment.

File: Code/Java/Diameter_BinaryTree.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        root.right.left = new Node(5);
+        root.right.right = new Node(6);
+        root.right.left.left = new Node(7);
+        root.right.left.right = new Node(8);
+ 
+        System.out.print("The diameter of the tree is " + getDiameter(root));
+    }
+}
+
+
```
Comment (krishna-NIT) [inline Code/Java/Diameter_BinaryTree.java:65]: add space complexity

File: Code/Java/Diameter_BinaryTree.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
+// The diameter of a tree(sometimes called the width) is the number of
+// nodes on the longest path between two end nodes.
+
```
Comment (krishna-NIT) [inline Code/Java/Diameter_BinaryTree.java:3]: Explain in brief the approach

Comment (nidhivanjare) [reply Code/Java/Diameter_BinaryTree.java:3]: okay !!

**Comment by krishna-NIT** [comment]
@rish-singhal Help here ,
There are 2 language one issue, it out of my thing, is it allowed in one pr, Please guide her from here

**Comment by rish-singhal** [comment]
@nidhivanjare it is not allowed to push 2 languages solution together.

~ it's better to do it in 2 PR, you will get double the points.

**Comment by nidhivanjare** [comment]
Okay I will create two PR's
Thanks for the guidance

**Comment by nidhivanjare** [comment]
DONE!!

**Comment by krishna-NIT** [comment]
I will Suggest to you that you should close this or and make a new clean pr 
only one issue in 1 pr
only 2 files need to be charged i.e code file and readme @nidhivanjare

**Comment by nidhivanjare** [comment]
You want me to create a new 2 PR's for the 1788 issue right in java and python?
I have already created a new PR for java. 
Do you want me to do it for python as well?

@rish-singhal @krishna-NIT

**Pull request closed without merging**